src/pages/TxSender.vue

```diff
@@ -3,12 +3,15 @@
 		<a-row>
 			<a-col :xs="{ span: 24 }" :lg="{ span: 10 }">
 				<h1>Transaction Input</h1>
+				<!-- TODO: Currently disabled for JSON -->
 				<a-switch
-					v-model:checked="isJsonSelected"
-					checked-children="json"
-					un-checked-children="msgpack"
+					v-model:checked="isMsgPackSelected"
+					checked-children="Base64 Msgpack"
+					un-checked-children="JSON"
+					:disabled="true"
 				/>
 				<a-textarea
+					placeholder="Base64 msgpack or JSON object"
```

Comment:
Depend on your select format we can show something like "Base64 msgpack" or "JSON".